charts/rasa-x/values.yaml

```
@@ -140,6 +140,17 @@ app:
   port: 5055
   # resources which app is required / allowed to use
   resources:
+  # envs is a dict containing app specific env variables
+  envs:
+  #  - name: DATABASE_URL
+  #    valueFrom:
+  #      secretKeyRef:
+  #        name: app-secret
+  #        key: database_url
+  # paths specifies the webhook and health check url paths of the action server app
+  paths:
+    webhook: /webhook
```

this is a bit more consistent to how it's currently named in `endpoints.yml`. What do you think?
```suggestion
    actionEndpointUrl: "/webhook"
```

---

Yup, that sounds good.  I'll make that change.